frontend/apps/crates/components/src/module/_groups/cards/edit/state.rs

```diff
@@ -11,7 +11,7 @@ use futures_signals::{
 use shared::domain::jig::{
     module::{
         body::{
-            Background, BodyExt, Image, Instructions, ThemeChoice,
+            Background, BodyExt, Image, Instructions,
             _groups::cards::{self as raw, BaseContent, Mode, Step},
```

Comment:
To resolve the conflict on this file:

```suggestion
            Background, BodyExt, Image, Instructions,
            _groups::cards::{self as raw, BaseContent, Mode, Step}, Audio
```